README.md

```diff
@@ -103,7 +103,7 @@ The Vonage Video API Reference App for React is currently supported on the lates
 - ![Safari icon](/docs/assets/safari.svg) Safari
 - ![Electron icon](/docs/assets/electron.svg) Electron
 
-*Note:* Mobile web views have limited supported at the moment.
+*Note:* Mobile web views have limited supported at the moment. The minimum supported device width is `360px`.
```

Comment:
This is the width of the S8+

Reply:
Nice! Are we also going to enforce this so that chrome cannot go smaller?

Reply:
After some mobbing, we decided this was a dev-tools issue.